src/pudl/package_data/eia861/column_maps/short_form_eia861.csv

```diff
@@ -4,13 +4,13 @@ utility_id_eia,1,1,1,1,1,1,1,-1,1,1,1
 utility_name_eia,2,2,2,2,2,2,2,-1,2,2,2
 entity_type,-1,-1,-1,3,3,3,3,-1,3,3,3
 state,3,3,3,4,4,4,4,-1,4,4,4
-ba_code,-1,4,4,5,5,5,5,-1,5,5,5
-total_revenue,4,5,5,6,6,6,6,-1,6,6,6
-total_sales,5,6,6,7,7,7,7,-1,7,7,7
-total_customers,6,7,7,8,8,8,8,-1,8,8,8
+balancing_authority_code_eia,-1,4,4,5,5,5,5,-1,5,5,5
+sales_revenue,4,5,5,6,6,6,6,-1,6,6,6
```

Comment:
A note on this `sales_revenue` column -- it's reported in 1000s of dollars which we don't want (all values should be reported in $1). In the transform function, use the `_thousand_to_one` function in the EIA861 transform module for this.